config/install/oe_authentication.settings.yml

```diff
@@ -3,3 +3,4 @@ register_path: 'eim/external/register.cgi'
 validation_path: 'TicketValidationService'
 assurance_level: TOP
 ticket_types: SERVICE,PROXY
+redirect_user_register_route: true
```

Comment:
Do we need an update path for this? In the route subscriber, it does nothing if the setting is not set and this is set to true to maintain the default functionality.

Reply:
It's just config management. When the config import will find a diff between installed `oe_authentication.settings` and the code version, will apply the code version, right?

Reply:
That would assume that a site or a distribution imports all settings with every update and also handle config sync conflicts. Normally you would expect that a new version of a module that has new settings will take care on its own to perform the necessary changes in the database.

Reply:
Ok, added an update path.